vagrant.go

```diff
@@ -174,6 +187,20 @@ func (v *Vagrant) Setup(start bool, env string, numNodes int) error {
 	return nil
 }
 
+// Setup initializes a vagrant testbed.
+func (v *Vagrant) Setup(args ...interface{}) error {
```

Comment:
sorry, so sorry to do this again. I realize now that if we passed configuration structs instead of options, that we would not need these interface casts, just validations of the data structure. I think it would result in better errors as well.

Do you agree?